src/JWT/IBase64UrlEncoder.cs

```diff
@@ -0,0 +1,9 @@
+namespace JWT
+{
+    public interface IBase64UrlEncoder
+    {
+        string UrlEncode(byte[] input);

```

Comment:
Seems a bit redundant to have `Url` twice, perhaps the method could just be `Encode` and `Decode`?